lib/resources/dstu2/medication_statement.yaml

```diff
@@ -174,7 +161,7 @@ fields:
               "start": "2014-11-03T14:38:00.000-05:00"
             }
           }
-          
+
           {
```

Comment:
can you split these into multi-examples? reference: https://github.com/cerner/fhir.cerner.com/blob/6bbed7b93178cdd00866c5abd685770a48a58fd3/lib/resources/r4/encounter.yaml#L65

Reply:
[split into a second example](https://github.com/cerner/fhir.cerner.com/pull/374/commits/6fb0b4a5a1bdcd0b07e086f542d44ded83b12430?file-filters%5B%5D=.yaml#diff-5321e3dd849f4c21936b5a38c0bf2092R165)